contracts/extensions/VotingReputation.sol

```diff
@@ -295,7 +295,8 @@ contract VotingReputation is DSMath, PatriciaTreeProofs {
       _vote == YAY &&
       !motion.escalated &&
       motion.stakes[YAY] == requiredStake &&
-      getSig(motion.action) == CHANGE_FUNCTION
+      getSig(motion.action) == CHANGE_FUNCTION &&
+      (motion.target == address(0x0) || colonyNetwork.isColony(motion.target))
```

Comment:
Surely we need the same logic in the equivalent checks in `finalizeMotion`? We should have a test targeting an expenditure in another colony.

Reply:
Sure, happy to add.